app/templates/base_root.html

```diff
@@ -12,6 +12,7 @@
     <link rel="stylesheet" href="{{ url_for('static',filename='_css/bootstrap.min.css') }}">
     <link rel="stylesheet" href="{{ url_for('static',filename='_css/now-ui-kit.css') }}">
     <link rel="stylesheet" href="{{ url_for('static',filename='_css/style.css') }}">
+    <link rel='stylesheet prefetch' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.5.0/css/font-awesome.min.css'>
```

Comment:
Please download fontawesome and put it into our static folder, so we do not rely on any cdns